test/isochrone.cc

```diff
@@ -123,31 +123,31 @@ void try_isochrone(loki_worker_t& loki_worker,
2.105574],[5.099663,52.107595],[5.102321,52.106670],[5.104635,52.107623],[5.107321,52.109955],[5.107983,52.107599],[5.110531,52.108147],[5.111321,52.104524],[5.115321,52.106504],[5.129321,52.104532],[5.131321,52.107349],[5.134213,52.106937],[5.133321,52.105371],[5.140321,52.108765],[5.141321,52.106324],[5.143963,52.108937],[5.142321,52.111219],[5.147970,52.113288],[5.144821,52.119937],[5.146321,52.122254],[5.149527,52.117937],[5.148684,52.113300],[5.154033,52.104649],[5.161819,52.104937],[5.153648,52.102937],[5.159648,52.093264],[5.163074,52.091937],[5.161802,52.079456],[5.151959,52.070937],[5.158932,52.066548],[5.159112,52.064937],[5.156739,52.064355],[5.152321,52.068117],[5.149126,52.065937],[5.149182,52.063798],[5.151701,52.063937],[5.152162,52.058778],[5.160004,52.058937],[5.155123,52.057135],[5.155145,52.055761],[5.159029,52.054937],[5.156139,52.054119],[5.155792,52.051466],[5.145104,52.052154],[5.142321,52.048204],[5.141321,52.050285],[5.124321,52.048191],[5.120321,52.049163],[5.118321,52.046164],[5.114893,52.046509],[5.113321,52.049186],[5.112321,52.046219],[5.111485,52.050101],[5.107570,52.051186],[5.104099,52.050159],[5.101182,52.051076],[5.098321,52.046251],[5.094321,52.048197],[5.093321,52.046314],[5.090321,52.047264],[5.089321,52.046308],[5.072978,52.046594],[5.066186,52.051072],[5.066428,52.052044],[5.069750,52.051508],[5.071012,52.057246],[5.073608,52.058224],[5.069968,52.058290],[5.068263,52.054879],[5.067760,52.057376],[5.065639,52.057937],[5.067613,52.061937],[5.064688,52.063304],[5.063321,52.066206],[5.033321,52.071573],[5.029839,52.071419],[5.025209,52.067937],[5.026952,52.071568],[5.025321,52.073406],[5.021982,52.073598],[5.022061,52.075197],[5.030321,52.074810],[5.029369,52.075937],[5.034141,52.077937],[5.035321,52.073258],[5.042771,52.072487],[5.043259,52.075937],[5.048321,52.081429],[5.047708,52.078324],[5.051237,52.075937],[5.050321,52.072423],[5.053321,52.077735],[5.052834,52.074450],[5.055660,52.074598],[5.054992,52.077937],[5.057797,52.078461],[5.059321,52.081150],[5.060329,52.078937],[5.054919,52.070937],[5.060321,52.069593],[5.067613,52.070937],[5.066321,52.075319],[5.062321,52.076429],[5.059287,52.073903],[5.061002,52.077256],[5.069321,52.075623],[5.071556,52.076937],[5.071321,52.081586],[5.054198,52.081937],[5.050907,52.083937],[5.051799,52.085415],[5.049795,52.085463],[5.049321,52.083135],[5.048774,52.085390],[5.045321,52.085851],[5.047321,52.087985],[5.048321,52.086876],[5.054985,52.088273],[5.058046,52.090212],[5.058693,52.092937],[5.063321,52.092392],[5.063768,52.093937],[5.062321,52.096482],[5.060270,52.094886],[5.057742,52.096937],[5.059005,52.098621],[5.057204,52.098937],[5.060839,52.098937],[5.060634,52.103937],[5.058321,52.102771],[5.053207,52.103823],[5.049845,52.106937],[5.049421,52.111037],[5.034321,52.116532],[5.029321,52.113579],[5.031766,52.118382],[5.026321,52.120692],[5.023255,52.123937],[5.033321,52.118969],[5.040321,52.125974]],"type":"LineString"},"type":"Feature"}],"type":"FeatureCollection"})";
```

Comment:
the only changes here is that now in the properties we serialize which type of contour it was time or distance